qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -181,6 +182,21 @@ class FixedFrequencyTransmon(BasisGateLibrary):
 
     Note that for now this library supports single-qubit gates and will be extended
     in the future.
+
+    Provided gates:
+        - x: :math:`\pi` pulse around the x-axis.
+        - sx: :math:`\pi/2` pulse around the x-axis.
+        - y: :math:`\pi` pulse around the y-axis.
+        - sy: :math:`\pi/2` pulse around the y-axis.
+
+    Parameters:
+        - duration: Duration of the pulses.
+        - σ: Standard deviation of the pulses.
+        - amp: Amplitude of the pulses.
```

Comment:
Maybe this could be more succinct:

```suggestion
        - amp: Amplitude of the ``x`` and ``y``  pulses (``sx`` and ``sy`` use ``amp / 2``).
```

Reply:
Hmm, maybe the point I was making needs to go in the default_values docstring. I was trying to clarify that the default value `amp` sets the x/y amp and the sx/sy get set to half that value.

Reply:
Done here: [32ab306](https://github.com/Qiskit/qiskit-experiments/pull/889/commits/32ab30631e901210a7b38cc51e1497b5eff6154c)